connections/src/app/app.component.html

```diff
@@ -1,3 +1,5 @@
+<!-- this is to check which environment we're in currently -->
+<div class="text-center font-bold">This app is running in the {{ name }}.</div>
```

Comment:
???

Reply:
This is just to know that what environment we're using so that it's visible.
If you want we can remove it as well.

![image](https://github.com/michael-gokey-architect/connections.biotool.me/assets/62620333/27f521c3-cd74-4867-9f8c-517bd6958815)